app/web/features/search/MapWrapper.tsx

```diff
@@ -103,13 +110,17 @@ export default function MapWrapper({
   setIsFiltersOpen,
   wasSearchPerformed,
   setWasSearchPerformed,
+  areFiltersCleared,
+  onClearFiltersClick,
 }: mapWrapperProps) {
   const { t } = useTranslation([SEARCH]);
   const [areClustersLoaded, setAreClustersLoaded] = useState(false);
   const [isMapStyleLoaded, setIsMapStyleLoaded] = useState(false);
   const [isMapSourceLoaded, setIsMapSourceLoaded] = useState(false);
   const previousResult = usePrevious(selectedResult);
   const classes = useStyles();
+  const theme = useTheme();
+  const isMobile = useMediaQuery(theme.breakpoints.down("sm"));
```

Comment:
```suggestion
  const isMobile = useMediaQuery(theme.breakpoints.down("md"));
```

Medium here to represent mobile as it doesn't include the size defined. So this would mean "below medium" rather than "medium and below".